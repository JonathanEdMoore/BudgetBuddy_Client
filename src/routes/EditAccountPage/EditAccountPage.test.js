import React from 'react'
import ReactDOM from 'react-dom'
import EditAccountPage from './EditAccountPage'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('EditAccountPage smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><EditAccountPage /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('EditAccountPage snapshot test', () => {
  const wrapper = shallow(<EditAccountPage />)
  expect(toJson(wrapper)).toMatchSnapshot()
})