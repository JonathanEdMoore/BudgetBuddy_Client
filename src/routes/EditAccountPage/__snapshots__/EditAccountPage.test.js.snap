// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <main
    role="main"
  >
    <header>
      <MainHeader />
      <h2>
        Edit Account
      </h2>
    </header>
    <section>
      <BudgetBuddyForm
        onSubmit={[Function]}
      >
        <div
          className="field"
        >
          <label
            htmlFor="first-name"
          >
            Change First name
          </label>
          <input
            id="first-name"
            name="first-name"
            onChange={[Function]}
            placeholder="First Name"
            type="text"
          />
        </div>
        <div
          className="field"
        >
          <label
            htmlFor="last-name"
          >
            Change Last name
          </label>
          <input
            id="last-name"
            name="last-name"
            onChange={[Function]}
            placeholder="Last Name"
            type="text"
          />
        </div>
        <button
          disabled={true}
          type="submit"
        >
          Update
        </button>
        <Link
          to="/delete_account"
        >
          <button>
            Delete Account
          </button>
        </Link>
      </BudgetBuddyForm>
    </section>
  </main>
</div>
`;
